src/helper/UserSession.cpp

```diff
@@ -110,6 +111,22 @@ namespace SDDM {
             }
         }
 
+        // enter Linux namespaces
```

Comment:
this block of code is linux specific, should be wrapped in a `#ifdef Q_OS_LINUX`

Reply:
Does SDDM support or plans to support other OSs?

Reply:
FreeBSD users might be interested in using SDDM but personally I'm not working on that goal.